Mosaicstg/climate-united
# Nav Dropdown

# Description

These changes add initial support for nav drop downs in the header navigation.

These changes also updates the following packages to their latest version:
- `@remix-run/*`
- `@netlify-functions/remix-adapter`
- `vite-env-only`
- `vite-tsconfig-paths`
- `framer-motion`
- `@radix-ui`
- `react`
- `@types/react`
- `tailwindcss`
- `zod`

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

1. Pull down latest code
2. Run `npm install` to install latest packages
3. Run dev server locally `npm run dev`
4. Verify that nav dropdown works by clicking the `About Us` nav item on desktop and mobile

# Checklist:
